Rubberduck.Parsing/Annotations/ExcelHotKeyAnnotation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Parsing.Annotations
+{
+    public sealed class ExcelHotKeyAnnotation : FlexibleAttributeValueAnnotationBase
+    {
+        public ExcelHotKeyAnnotation(AnnotationType annotationType, QualifiedSelection qualifiedSelection, VBAParser.AnnotationContext context, IEnumerable<string> attributeValues) :
```

Comment:
This will not work.
The annotation type does not belong into the ctor for the concrete annotation; pass the expected one to the base class instead. 

The annotation are created via reflection and, thus, all have to have compatible signatures.

Reply:
The last parameter has a misleading name: you will get the annotation values there, not the attribute values, if they differ.